arrayMethods_lab.js

```diff
@@ -0,0 +1,291 @@
+// # Exercises
+//
+// 1. Given an array of numbers, write a function that returns a new array where every element is incremented by 10.
+//
+// ```
+// Input: [1,2,3,4,5]
+// Output: [11,12,13,14,15]
+// ```
+console.log('----- Question 1 -----');
+let arr1 = [1, 2, 3, 4, 5];
+arr1.forEach((element) => {
+  console.log(element + 10);
+});
+
+// 2. Given an array, return a new array where every non-string element is converted to an empty string.
+//
+// ```
+// Input: ["a", 123, "b", "c", {name: "cat"}]
+// Output: ["a", "", "b", "c", ""]
+// ```
+console.log('----- Question 2 -----');
+let arr2 = ['b', 4, 'c', null, 'd', { animal: 'cheetah' }];
+let onlyStrings = arr2.map((currentValue) => {
+  if (typeof (currentValue) !== typeof ('string')) {
+    return '';
+  } else {
+    return currentValue;
+  }
+});
+console.log(onlyStrings);
+
+// 3. Write a function that returns only the even elements in an array of numbers.
+//
+// ```
+// Input: [1,2,3,4,5]
+// Output: [2,4]
+// ```
+console.log('----- Question 3 -----');
+let arr3 = [1, 2, 3, 4, 5];
+let evenArray = arr3.filter((currentValue) => {
+      return currentValue % 2 === 0;
+    });
+console.log(evenArray);
+
+// 4. Write a function that checks if all elements in the array are the same.
+//
+// ```
+// Input: [1,1,1,1,1]
+// Output: true
+//
+// Input: [1,1,1,1,2]
+// Output: false
+// ```
+console.log('----- Question 4 -----');
+let arr4 = ['b', 'b', 'b', 'b', 'm'];
+let allSame = arr4.every((currentValue) => {
+  let index1 = arr4[0];
+  if (currentValue === index1) {
+    return true;
+  }
+
+  return false;
+});
+console.log(allSame);
+
```

Comment:
Good job, remember the every method returns a boolean. so you don't need those returns.